client/src/components/ReplyDialog.js

```diff
@@ -59,14 +62,24 @@ function ReplyDialog(props) {
     setBlobObject(recordedBlob);
   };
 
-  function handleSubmit() {
+  async function handleSubmit() {
+    //get current user ID 
+    const userID = window.localStorage.getItem("userID");
```

Comment:
You shouldn't need to do this I believe, the userID is stored in this.props.location.state.id of feed.js page and you can pass that down to these children components as an id prop